doc/cephadm/services/monitoring.rst

```diff
@@ -177,20 +177,33 @@ To see the default container images, run a command of the following form:
 
 .. prompt:: bash #
 
-   grep -E "DEFAULT*IMAGE" /var/lib/ceph/{FSID}/cephadm.{DIGEST}
+   cephadm list-images
 
 ::
 
-   DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.51.0'
-   DEFAULT_LOKI_IMAGE = 'docker.io/grafana/loki:2.9.5'    
-   DEFAULT_PROMTAIL_IMAGE = 'docker.io/grafana/promtail:2.9.5'    
-   DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v1.7.0'    
-   DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.27.0'   
-   DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/grafana:10.4.0'
+   [mgr]
+   mgr/cephadm/container_image_prometheus = quay.io/prometheus/prometheus:v2.51.0
+   mgr/cephadm/container_image_loki = docker.io/grafana/loki:3.0.0
+   mgr/cephadm/container_image_promtail = docker.io/grafana/promtail:3.0.0
+   mgr/cephadm/container_image_node_exporter = quay.io/prometheus/node-exporter:v1.7.0
+   mgr/cephadm/container_image_alertmanager = quay.io/prometheus/alertmanager:v0.27.0
+   mgr/cephadm/container_image_grafana = quay.io/ceph/grafana:10.4.8
+   mgr/cephadm/container_image_haproxy = quay.io/ceph/haproxy:2.3
+   mgr/cephadm/container_image_keepalived = quay.io/ceph/keepalived:2.2.4
+   mgr/cephadm/container_image_nvmeof = quay.io/ceph/nvmeof:1.2.17
+   mgr/cephadm/container_image_snmp_gateway = docker.io/maxwo/snmp-notifier:v1.2.1
+   mgr/cephadm/container_image_elasticsearch = quay.io/omrizeneva/elasticsearch:6.8.23
+   mgr/cephadm/container_image_jaeger_collector = quay.io/jaegertracing/jaeger-collector:1.29
+   mgr/cephadm/container_image_jaeger_agent = quay.io/jaegertracing/jaeger-agent:1.29
+   mgr/cephadm/container_image_jaeger_query = quay.io/jaegertracing/jaeger-query:1.29
+   mgr/cephadm/container_image_samba = quay.io/samba.org/samba-server:devbuilds-centos-amd64
+   mgr/cephadm/container_image_samba_metrics = quay.io/samba.org/samba-metrics:latest
+   mgr/cephadm/container_image_nginx = quay.io/ceph/nginx:sclorg-nginx-126
+   mgr/cephadm/container_image_oauth2_proxy = quay.io/oauth2-proxy/oauth2-proxy:v7.6.0
+    
```

Comment:
I think this is really hard to maintain manually and since we are introducing a new class `DefaultImages` we should use sphinx autodoc for this. Plz, have a look to examples in other files such as mgmt-gateway.rst:

```
.. autoclass:: MgmtGatewaySpec
   :members:
```
In this case some customization is needed to generate the format we want but I think it should be feasible.

Reply:
Instead of adding here statically, i am creating a new rst file with images dynamically and including file here. Let me know if it looks good.

Reply:
![image](https://github.com/user-attachments/assets/f9b7ed9a-5e1d-4e85-8b3b-7fb220dfbad9)

Reply:
that looks really good @ShwetaBhosale1 thanks :+1: